src/main/resources/plugin.yml

```diff
@@ -1,8 +1,8 @@
 name: ${project.artifactId}
 version: ${project.version}
 authors: [valadian, rourke750, ProgrammerDan, Maxopoly, DeadBeef, Protonull, Okx, mindless728, Tealnerd]
-main: ${plugin.main}
-api-version: ${minecraft.version}
+main: ${project.groupId}.${project.artifactId}Plugin
```

Comment:
This should just be hardcoding the name, because doing it this way provides no advantage, but has the downside of possibly breaking in the future. I know it was already like that before, but same thing.